styles/main.css

```diff
@@ -9,19 +9,56 @@ html {
 body {
     font-family: "Open Sans", "Helvetica", sans-serif;
     line-height: 1.5;
+    background: url("../images/atomist-starry-night.jpg") no-repeat center top;
     background-color: #1b2937;
 }
```

Comment:
I would suggest adding `background-size: 100% auto;` here to let the background stretch to the full width on viewports larger than 1680px.